cmd/storage-rest-server.go

```diff
@@ -284,9 +285,9 @@ func (s *storageRESTServer) ReadAllHandler(w http.ResponseWriter, r *http.Reques
 		s.writeErrorResponse(w, err)
 		return
 	}
-	w.Header().Set(xhttp.ContentLength, strconv.Itoa(len(buf)))
-	w.Write(buf)
-	w.(http.Flusher).Flush()
+	lw := lz4.NewWriter(w)
```

Comment:
@klauspost actually this PR should be closed, this PR was tried as a way to increase the internode traffic bandwidth. But with input stream coming in as encrypted when we use server-side encryption. I thought it may be an overkill to add this behavior.

Reply:
Too bad. With a decent CPU and a concurrent implementation is Snappy or LZ4, it should be fast enough to saturate a 10Gb ethernet.

Reply:
It must be, I will still keep this PR handy in-branch in-case we need to test theories.